api/ws/rpc/wsRPC.js

```diff
@@ -181,8 +192,17 @@ ClientRPCStub.prototype.sendAfterSocketReadyCb = function (connectionState) {
 	};
 };
 
+var remoteAction = function () {
+	throw new Error('Function invoked on a master instead of slave process');
```

Comment:
Function invoked on master instead of slave process

Should only be one master yes?